### PR TITLE
ROS2 Linting: spline_interpolation

### PR DESCRIPTION
**Pull request by jilaada**
## Summary

Simple lint of the `spline_interpolation` package. Only changes were in package.xml where I fixed the indentation.

## Testing
1. Compile with the correct build flags
```
colcon build --packages-up-to spline_interpolation --cmake-args -DCMAKE_EXPORT_COMPILE_COMMANDS=1
```
2. Run the linter tests
```
colcon test --packages-select spline_interpolation && colcon test-result --verbose
```
3. Run clang-tidy on the the source files from the root AutowareArchitectureProposal directory
```
clang-tidy -p build/compile_commands.json src/autoware/autoware.iv/common/math/spline_interpolation/src/*
clang-tidy -p build/compile_commands.json src/autoware/autoware.iv/common/math/spline_interpolation/include/spline_interpolation/*
```

### CODE REVIEW

File: common/math/spline_interpolation/package.xml [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
 <?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="3">
-<name>spline_interpolation</name>
-<version>0.1.0</version>
-<description>The spline interpolation package</description>
-<maintainer email="fumiya.watanabe@tier4.jp">Fumiya Watanabe</maintainer>
-<author email="fumiya.watanabe@tier4.jp">Fumiya Watanabe</author>
-  <license>Apache 2</license>
-<buildtool_depend>ament_cmake_auto</buildtool_depend>
+	<name>spline_interpolation</name>
```
Comment (esteve) [inline common/math/spline_interpolation/package.xml:4]: @jilaada could you revert the spaces-to-tabs changes? Thanks.

**Review by esteve — CHANGES_REQUESTED** [review]

**Comment by esteve** [comment]
@jilaada I've replaced 4-space indents with 2 spaces to match the other `package.xml` files in https://github.com/tier4/Pilot.Auto/pull/203/commits/4614be6aa04e349d80a6ffbb8210a462193d782b

**Review by esteve — APPROVED** [review]

**Comment by jilaada** [comment]
> @jilaada I've replaced 4-space indents with 2 spaces to match the other `package.xml` files in [4614be6](https://github.com/tier4/Pilot.Auto/commit/4614be6aa04e349d80a6ffbb8210a462193d782b)

Ah thanks for making that change!

**Comment by esteve** [comment]
@jilaada no problem! :slightly_smiling_face:

**Pull request merged**